client/src/annoMatrix/annoMatrix.ts

```diff
@@ -641,10 +656,7 @@ Return cache keys for columns associated with this query.  May return
     }
   }
 
-  _gcFetchCleanup(
-    field: Field,
-    pinnedColumns: string[] | number[] | [undefined]
-  ): void {
+  _gcFetchCleanup(field: Field, pinnedColumns: string[] | number[]): void {
```

Comment:
This should return the new Dataframe LabelArray type (see my comment on the definition of WhereCacheColumnLabels)

Reply:
Updated [annoMatrix.ts:_gcFetchCleanup](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L656).